tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -958,94 +956,61 @@ describe('migrate-converged-pkg generator', () => {
     }
 
     it(`should setup .babelrc.json`, async () => {
-      const babelMakeStylesPkg = getScopedPkgName(tree, 'babel-make-styles');
       const projectConfig = readProjectConfiguration(tree, options.name);
 
-      let packageJson = getPackageJson(projectConfig);
-      let devDeps = packageJson.devDependencies || {};
-      expect(devDeps[babelMakeStylesPkg]).toBe(undefined);
-
       await generator(tree, options);
-
       let babelConfig = getBabelConfig(projectConfig);
 
       expect(babelConfig).toEqual({
-        plugins: [
-          'module:@fluentui/babel-make-styles',
-          'annotate-pure-calls',
-          '@babel/transform-react-pure-annotations',
-        ],
+        presets: ['@griffel'],
+        plugins: ['annotate-pure-calls', '@babel/transform-react-pure-annotations'],
       });
 
       tree.delete(`${projectConfig.root}/.babelrc.json`);
 
       await generator(tree, options);
-
       babelConfig = getBabelConfig(projectConfig);
-      packageJson = getPackageJson(projectConfig);
-      devDeps = packageJson.devDependencies || {};
 
       expect(babelConfig).toEqual({
-        plugins: [
-          'module:@fluentui/babel-make-styles',
-          'annotate-pure-calls',
-          '@babel/transform-react-pure-annotations',
-        ],
+        presets: ['@griffel'],
+        plugins: ['annotate-pure-calls', '@babel/transform-react-pure-annotations'],
       });
-      expect(devDeps[babelMakeStylesPkg]).toBe('9.0.0-alpha.0');
```

Comment:
We don't need to have Babel plugin in `devDependencies` of the package, that's why the test was trimmed a lot 🐱